mock-core-configs/etc/mock/templates/fedora-eln.tpl

```diff
@@ -0,0 +1,70 @@
+config_opts['root'] = 'fedora-eln-{{ target_arch }}'
+
+config_opts['chroot_setup_cmd'] = 'install @build'
+
+config_opts['dist'] = 'eln'  # only useful for --resultdir variable subst
+config_opts['extra_chroot_dirs'] = [ '/run/lock', ]
+config_opts['package_manager'] = 'dnf'
+config_opts['bootstrap_image'] = 'fedora:latest'
+
+config_opts['dnf.conf'] = """
+[main]
+keepcache=1
+debuglevel=2
+reposdir=/dev/null
+logfile=/var/log/yum.log
+retries=20
+obsoletes=1
+gpgcheck=0
+assumeyes=1
+syslog_ident=mock
+syslog_device=
+install_weak_deps=0
+metadata_expire=0
+best=1
+
+# TODO
+module_platform_id=platform:eln
+protected_packages=
+
+# repos
+[local]
+name=local
+baseurl=https://kojipkgs.fedoraproject.org/repos/eln-build/latest/$basearch/
+cost=2000
+enabled=0
+skip_if_unavailable=False
+
+[eln]
+name=Fedora ELN - Developmental modular packages for the next Enterprise Linux release
+#metalink=https://mirrors.fedoraproject.org/metalink?repo=eln&arch=$basearch
+baseurl=https://odcs.fedoraproject.org/composes/production/latest-Fedora-ELN/compose/Everything/$basearch/os/
```

Comment:
@sgallagher I'm not able to install dnf from here.  Is this just a Rawhide addon (meaning that we are missing rawhide repo here)?

Reply:
https://odcs.fedoraproject.org/composes/production/latest-Fedora-ELN/compose/Everything/x86_64/os/Packages/d/dnf-4.2.23-2.eln103.noarch.rpm exists, so I'm not sure what you mean.

Is it missing a dependency?

Reply:
> Is it missing a dependency?

Yes:
```
Error: 
 Problem 1: package dnf-4.2.23-2.eln103.noarch requires python3-dnf = 4.2.23-2.eln103, but none of the providers can be installed
  - package python3-dnf-4.2.23-2.eln103.noarch requires dnf-data = 4.2.23-2.eln103, but none of the providers can be installed
  - conflicting requests
  - nothing provides libreport-filesystem needed by dnf-data-4.2.23-2.eln103.noarch
 Problem 2: package python3-dnf-plugins-core-4.0.16-4.eln103.noarch requires python3-dnf >= 4.2.22, but none of the providers can be installed
  - package dnf-plugins-core-4.0.16-4.eln103.noarch requires python3-dnf-plugins-core = 4.0.16-4.eln103, but none of the providers can be installed
  - package python3-dnf-4.2.23-2.eln103.noarch requires dnf-data = 4.2.23-2.eln103, but none of the providers can be installed
  - conflicting requests
  - nothing provides libreport-filesystem needed by dnf-data-4.2.23-2.eln103.noarch
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
```

Reply:
ping

Reply:
The situation with missing dependencies is better now and `dnf` installs.

Now mock init fails with:
```
Module or Group 'build' is not available.
```

Reply:
It appears `fedora-eln.tpl` just needs a slight fix to work:
```
-config_opts['chroot_setup_cmd'] = 'install @build'
+config_opts['chroot_setup_cmd'] = 'install @buildsys-build'
```

Reply:
Indeed, it just works now.  Thank you for the ping.